src/blumkin/cli.py

```
@@ -1768,5 +1770,55 @@ def meeting_transcription_cmd(
+        payload = asyncio.run(people_resolve(name=name, email=email, top=top))
+    except LookupError as exc:
+        emit_error(error="not_found", message=str(exc), as_json=as_json)
+        raise SystemExit(EXIT_NOT_FOUND) from exc
+    except ValueError as exc:
+        msg = str(exc)
+        if "client_id" in msg or "Missing" in msg:
+            emit_error(error="auth_required", message=msg, as_json=as_json)
+            raise SystemExit(EXIT_AUTH) from exc
+        emit_error(error="usage_error", message=msg, as_json=as_json)
+        raise SystemExit(EXIT_USAGE) from exc
+    except Exception as exc:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)